pkg/controller/controller.go

```diff
@@ -435,17 +448,20 @@ func (c *Controller) runAction(ctx context.Context, t *tomb.Tomb, as *crv1alpha1
 	c.logAndSuccessEvent(ctx, fmt.Sprintf("Executing action %s", action.Name), "Started Action", as)
 	tp, err := param.New(ctx, c.clientset, c.dynClient, c.crClient, c.osClient, action)
 	if err != nil {
+		c.incrementActionSetResolutionCounterVec(ACTION_SET_COUNTER_VEC_LABEL_RES_FAILURE)
```

Comment:
Could you remind me what the issue was with incrementing this in the caller of `runAction` i.e., `handleActionSet` inside the `if` block on line 421?

Reply:
@pavannd1 There is no issue per se with respect to this PR. Considering that we do move the `Inc()` calls to line 421 now, and we decide to pair the `resolution` label dimension with another dimension, say `action_type` in future PRs, we would have to move back this `Inc()` from line 421 to the callee, because at line 421, we don't have the `action_type` information.  Hence, I thought that it would be better to maintain the `Inc()` calls in the callee to avoid make an additional change when we add `action_type` dimension. 

I think this would apply to any other label dimension we choose to use with the `resolution` dimension.